api_session.yaml

```diff
@@ -8,6 +8,35 @@ basePath: /api/v1
 schemes:
   - https
 paths:
+  "/sessions/{fiscalcode}":
+    get:
+      operationId: getSession
+      summary: Get available user info from session
+      description: |-
+        Use this operation if you want to known the state of user session
+      parameters:
+        - $ref: "#/parameters/FiscalCode"
+        - $ref: "#/parameters/Token"
+      responses:
+        "200":
+          description: Success.
+          schema:
+            $ref: "#/definitions/UserSession"
+          examples:
+            application/json:
+              "active": "true"
+        "400":
+          description: Bad request
+        "401":
+          description: Token null or invalid.
+        "404":
+          description: User not found.
+        "429":
```

Comment:
Where is configured the Throttling?

Reply:
nowhere (for now). I added it for future implementation.